packages/playground/src/router/index.ts

```diff
@@ -122,6 +122,11 @@ const router = createRouter({
       component: () => import("../views/contracts_list.vue"),
       meta: { title: "Contracts List", info: { page: "info/contracts_list.md" } },
     },
+    {
+      path: "/twin",
+      component: () => import("../portal/twin_view.vue"),
+      meta: { title: "Twin", info: { page: "info/twin.md" } },
```

Comment:
There's no twin.md created and I don't think we need one. remove the info object.